test/integration/smoke/test_templates.py

```diff
@@ -735,8 +735,7 @@ def test_03_delete_template(self):
 
         list_template_response = Template.list(
                                     self.apiclient,
-                                    templatefilter=\
-                                    self.services["templatefilter"],
+                                    templatefilter='selfexecutable',
```

Comment:
alarm bells are ringing in my ear when I see this, can you explain @Spaceman1984 ? we are no longer filtering according to the configured filter, but now hardcoding the filter type?!?!

Reply:
@DaanHoogland this test needed to be modified because the behaviour of the "self" filter had to be changed in order to see inactive system VM templates from the UI, previously if a template was inactive it wouldn't be displayed on the UI at all. I had a look inside this test_templates.py file and I see no other way around hardcoding the filter.